src/Explore/Services.js

```diff
@@ -18,14 +18,7 @@ const multiChecker = (...urls) =>
   )
 
 const injectId = (id, template) => {
-  if (!template) {
-    return
-  }
-  if (!template.includes('%')) {
-    throw new Error('Invalid service template')
-  }
-  const [prefix, suffix] = template.split('%')
-  return suffix ? prefix + id + suffix : prefix + id
+  return template ? template.replace('__PID__', id) : undefined
```

Comment:
- I no longer throw an error to allow templates that don't include any injection point (since the search API doesn't return the information I need in order to know the proposal's path on the ESRF cluster).
- I also change `%` to `__PID__`, as `%` is used for encoding URL query params.

Reply:
That's way better, thanks!